### DISCUSSION TITLE
Download profiles

### DISCUSSION THREAD

**Discussion by benmathews** [grafana/phlare#458] [Development]
Phlare will never (nor should it) do everything possible to visualize and analyze the profile. Make it possible to download the profile for offline analysis.

**Comment by cyriltovena** [comment]
Great idea !

**Reply by cyriltovena** [reply]
We merged a PR to allow this from a CLI but we'll add this to Grafana cc @joey-grafana This should be easy to add now with the new API.

**Reply by joey-grafana** [reply]
Let's talk about this next week and see what we can do.
